astropy/wcs/wcs.py

```diff
@@ -902,10 +902,10 @@ def write_d2i(num, det2im):
             hdulist[0].header[f'{d_kw}{num:d}.EXTVER'] = (
                 num, 'Version number of WCSDVARR extension')
             hdulist[0].header[f'{d_kw}{num:d}.NAXES'] = (
-                len(det2im.data.shape), 'Number of independent variables in d2im function')
+                len(det2im.data.shape), 'Number of independent variables in D2IM function')
             for i in range(det2im.data.ndim):
                 hdulist[0].header['{}{:d}.AXIS.{:d}'.format(d_kw, num, i + 1)] = (
-                    i + 1, 'Axis number of the jth independent variable in a d2im function')
```

Comment:
I like the reduction of warning, but is "independent" really redundant here?

Also, will changing this trips DMS data verification step when we deliver pipeline products? (This is STScI specific and we can discuss offline if you wish.)